configs/bundle_config.yaml

```diff
@@ -15,126 +15,206 @@
+    output:
+      - data/costs.csv
+      - data/hydro_capacities.csv
+      - data/custom_powerplants.csv
+      - data/raw/eez/eez_v11.gpkg
+      - data/raw/gebco/GEBCO_2021_TID.nc
+      - data/raw/copernicus/PROBAV_LC100_global_v3.0.1_2019-nrt_Discrete-Classification-map_EPSG-4326.tif
+
+  # tutorial bundle specific for Nigeria and Benin only
+  bundle_resources_tutorial:
+    countries: [NG, BJ]
+    tutorial: true
+    category: resources
+    destination: "resources"
+    urls:
+      zenodo: https://sandbox.zenodo.org/record/1016001/files/tutorial_resources.zip?download=1
+      gdrive: https://drive.google.com/file/d/1WWVyXHA2E7aGcg_OC1lN8Pb2tA8DgfU0/view?usp=sharing
+    output:
+      - resources/ssp2-2.6/2030/era5_2013/Africa.nc
+      - resources/natura.tiff
+
+  # tutorial bundle specific for Nigeria and Benin only
+  bundle_cutouts_tutorial:
+    countries: [NG, BJ]
+    tutorial: true
+    category: cutouts
+    destination: "cutouts"
+    urls:
+      zenodo: https://sandbox.zenodo.org/record/1016015/files/tutorial_cutouts.zip?download=1
+      gdrive: https://drive.google.com/file/d/1Lro4X92sVuD30C0B3sQ2jJU37uZ50DWi/view?usp=sharing
+    output: [cutouts/africa-2013-era5-tutorial.nc]
+
+  # data bundle containing the data of the data folder common to all regions of the world
+  bundle_data_world:
+    countries: [world]
+    category: common
+    destination: "data"
+    urls:
+      zenodo: https://sandbox.zenodo.org/record/1016540/files/common_data.zip?download=1
+      gdrive: https://drive.google.com/file/d/1fo-YsyU2LKkVeg5Ea0JbcEx-zQ3kJKW7/view?usp=sharing
+    output:
+      - data/costs.csv
+      - data/hydro_capacities.csv
+      - data/custom_powerplants.csv
+      - data/raw/eez/eez_v11.gpkg
+      - data/raw/gebco/GEBCO_2021_TID.nc
+      - data/raw/copernicus/PROBAV_LC100_global_v3.0.1_2019-nrt_Discrete-Classification-map_EPSG-4326.tif
+
+
+  # resources bundle containing the resources folder for africa only
+  bundle_resources_world:
+    countries: [world]
+    category: resources
+    destination: "resources"
+    urls:
+      zenodo: https://sandbox.zenodo.org/record/1016878/files/resources_africa.zip?download=1
+      gdrive: https://drive.google.com/file/d/1abwx1FviogYUsnP27MrCzQf-3w-9IHDl/view?usp=sharing
+    output:
+      - resources/ssp2-2.6/2030/era5_2013/Africa.nc
+      - resources/natura.tiff
+
+  # data bundle of the data folder for the african continent
+  bundle_landcover_africa:
+    countries: [africa]
+    category: landcover
+    destination: "data/raw/landcover"
+    urls:
+      zenodo: https://sandbox.zenodo.org/record/1016042/files/landcover_africa.zip?download=1
+      gdrive: https://drive.google.com/file/d/1l6Xgr3PXyvgqWaXUbto9zvvMS_OgSjvG/view?usp=sharing
+    output: [data/raw/landcover/*]
+
+  # data bundle of the data folder for the african continent
+  bundle_landcover_asiapacific:
+    countries: [asia, australia-oceania]
```

Comment:
In the demand PR, I changed the naming of the continents and combined some regions to align with the ISO standard (i.e. central-america + north-america = NorthAmerica). This would require changes here as well. Let's discuss that during the next data & workflow meeting